frontend/src/components/core/Block/Block.tsx

```diff
@@ -216,18 +218,37 @@ class Block extends PureComponent<Props> {
       audio: (el: SimpleElement) => <Audio element={el} width={width} />,
       balloons: (el: SimpleElement) => <Balloons element={el} width={width} />,
       bokehChart: (el: SimpleElement) => (
-        <BokehChart element={el} index={index} width={width} />
+        <FullScreenWrapper width={width}>
+          {({ width, height }) => (
+            <BokehChart
+              element={el}
+              index={index}
+              width={width}
+              height={height}
+            />
+          )}
+        </FullScreenWrapper>
```

Comment:
Can you move these `FullScreenWrapper`s into each element's component?  It would be nice to keep this big switch statement lean.